package.json

```diff
@@ -90,23 +90,23 @@
 		"/dist"
 	],
 	"dependencies": {
-		"@oclif/command": "1.4.33",
-		"@oclif/config": "1.6.28",
-		"@oclif/errors": "1.1.2",
-		"@oclif/plugin-help": "2.0.5",
-		"babel-polyfill": "6.26.0",
-		"bip39": "2.4.0",
-		"chalk": "2.3.0",
-		"cli-table2": "LiskHQ/cli-table2#187c6ae80b9d963c598d0cb3379153387bfb15c4",
-		"inquirer": "6.0.0",
-		"lisk-elements": "1.0.0-rc.1",
-		"lockfile": "1.0.3",
-		"semver": "5.5.0",
+		"@oclif/command": "1.5.0",
+		"@oclif/config": "1.7.4",
+		"@oclif/errors": "1.2.0",
+		"@oclif/plugin-help": "2.1.1",
+		"babel-polyfill": "^6.26.0",
+		"bip39": "2.5.0",
+		"chalk": "2.4.1",
+		"cli-table2": "github:LiskHQ/cli-table2#187c6ae80b9d963c598d0cb3379153387bfb15c4",
```

Comment:
What's the benefit of this? Just being explicit?

Reply:
This was automatically changed by npm